std/encoding.d

```diff
@@ -2635,10 +2635,11 @@ abstract class EncodingScheme
  */
 class EncodingSchemeASCII : EncodingScheme
 {
+    /* // moved to std.internal.phobosinit
     shared static this()
     {
```

Comment:
I could if that's the consensus. The issue is that these really should go here. The example in the base class says to do it this way (and it will be fine for external libs to do that)